corehq/apps/userreports/expressions/__init__.py

```diff
@@ -2,6 +2,15 @@
 from django.utils.module_loading import import_string
 from corehq.apps.userreports.expressions.factory import ExpressionFactory
 
+
+def get_custom_ucr_expressions():
+    custom_ucr_expressions = settings.CUSTOM_UCR_EXPRESSIONS
+
+    for expression_list in settings.CUSTOM_UCR_EXPRESSION_LISTS:
+        custom_ucr_expressions += import_string(expression_list)
```

Comment:
I misnamed `expression_list` var. It is just a path that contains `settings.CUSTOM_UCR_EXPRESSIONS` like list

````python
In [11]: settings.CUSTOM_UCR_EXPRESSION_LISTS
Out[11]: 
['mvp.ucr.reports.expressions.CUSTOM_UCR_EXPRESSIONS',
 'custom.icds.ucr.expressions.CUSTOM_UCR_EXPRESSIONS']

In [12]: import_string(settings.CUSTOM_UCR_EXPRESSION_LISTS[0])
Out[12]: 
[('mvp_medical_cause', 'mvp.ucr.reports.expressions.medical_cause_expression'),
 ('mvp_no_treatment_reason',
  'mvp.ucr.reports.expressions.no_treatment_reason_expression'),
 ('mvp_treatment_provider_name',
  'mvp.ucr.reports.expressions.treatment_provider_name_expression'),
 ('mvp_treatment_place_name',
  'mvp.ucr.reports.expressions.treatment_place_name_expression'),
 ('mvp_death_place', 'mvp.ucr.reports.expressions.death_place_expression')]